WikiWatershed/model-my-watershed
# Disable Select Boundary If Below Min Zoom + Lower Boundary Min Zoom

## Overview

Before you were able to use the boundary selection draw tools even if the boundary layer you were selecting couldn't be accessed at the current zoom level. This PR fixes the disabling of the selection tools on zoom, closes out active tools if a zoom event makes them inaccessible, and decreases the min zoom for the boundary layers so they can be accessed at higher altitudes.

Connects #2217 

### Demo

![ch94ywf8xn](https://user-images.githubusercontent.com/7633670/30829417-b7b4e71c-a20e-11e7-8651-58f132606e99.gif)

## Testing Instructions

 * Pull this branch, and `bundle`
* Zoom around the map with select boundary tool open, and confirm layers are disabled when you zoom out passed their min zoom level
* Select a boundary tool and zoom way out — it should deselect when you zoom past its min zoom
